process/transactions/scDeploysProcessor.go

```diff
@@ -0,0 +1,232 @@
+package transactions
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/hex"
+
+	"github.com/ElrondNetwork/elastic-indexer-go/data"
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/hashing/keccak"
+	"github.com/ElrondNetwork/elrond-go/process/factory"
+)
+
+const (
+	delegationManagerAddress = "erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqylllslmq6y6"
+)
+
+type scDeploysProc struct {
+	pubKeyConverter      core.PubkeyConverter
+	scDeployReceiverAddr string
+	selfShardID          uint32
+}
+
+type deployerDto struct {
+	receiver string
+	sender   string
+	nonce    uint64
+	data     []byte
+	txHash   string
+}
+
+func newScDeploysProc(pubKeyConverter core.PubkeyConverter, selfShardID uint32) *scDeploysProc {
```

Comment:
maybe add nil checks?

Reply:
checks for nil are made when `txsDatabaseProcessor` is created